src/Kernel/Magic/Simulate.cs

```diff
@@ -28,7 +28,28 @@ public class SimulateMagic : AbstractMagic
         public SimulateMagic(ISymbolResolver resolver, IConfigurationSource configurationSource) : base(
             "simulate",
             new Documentation {
-                Summary = "Runs a given function or operation on the QuantumSimulator target machine"
+                Summary = "Runs a given function or operation on the QuantumSimulator target machine.",
+                Description = @"
+                    This magic command allows executing a given function or operation on the QuantumSimulator, 
+                    which performs a full-state simulation of the given function or operation
+                    and prints the resulting return value.
+
+                    #### Required parameters
+
+                    - Q# operation or function name. This must be the first parameter, and must be a valid Q# operation
+                    or function name that has been defined either in the notebook or in a Q# file in the same folder.
+                    - Arguments for the Q# operation or function must also be specified as `key=value` pairs.
+                ".Dedent(),
+                Examples = new []
+                {
+                    @"
+                        Simulate a Q# operation defined as `operation MyOperation(a : Int, b : Int) : Result`:
```

Comment:
Would an example without any inputs be helpful?

Reply:
Yes, I think so. Also on the other simulator commands.